client/src/js/services/messenger.js

```diff
@@ -27,7 +27,6 @@ angular.module('bhima.services')
 
 
     function trust(data) {
-      console.log('trusting ...', String(data));
```

Comment:
This is perhaps the most important change made in the recent history.  So glad to see this gone. :)